### DISCUSSION TITLE
302重定向是否会使ip变换频繁导致封控

### DISCUSSION THREAD

**Discussion by YCZ01111** [AlistGo/alist#7267] [Q&A]
302重定向是访问端直连网盘吧，这样会因为频繁变化ip导致封控吗？

**Comment by Simonzhang8** [comment]
有可能，具体得看网盘服务商了。
一般个人在家、单位这样的使用问题不大，但如果是做视频站下载站这类的很多各地ip访问的话就很危险，后者不建议用302模式
